### ISSUE TITLE
Escaping newlines in single-line strings

### ISSUE THREAD

**Issue by jiribenes** [effekt-lang/effekt#423]
Continuing the discussion from #410, do we actually want to be able to escape newlines like this:
```scala
val myString = "let's say I want to \
continue on the other line"      // ^ escapes a newline

// ~> myString = "let's say I want to continue on the other line"
```

Personally, I'm completely fine to forbid this, especially since we have multiple-line strings as of #410.

**Comment by marzipankaiser** [comment]
Additional comment: If we want this, this is broken on Windows.

**Comment by jiribenes** [comment]
This was resolved by the new lexer #435, such strings are no longer supported.
